graphql/openreader/src/opencrud/orderBy.ts

```diff
@@ -26,41 +26,45 @@ export function getOrderByMapping(model: Model, typeName: string): OpenCrud_Orde
         MAPPING_CACHE.set(model, cache)
     }
     if (cache[typeName]) return cache[typeName]
-    return cache[typeName] = buildOrderByMapping(model, typeName, 2)
+    return cache[typeName] = buildOrderByMapping(model, typeName, 2, false)
 }
 
 
-function buildOrderByMapping(model: Model, typeName: string, depth: number): OpenCrud_OrderBy_Mapping {
+function buildOrderByMapping(model: Model, typeName: string, depth: number, nullable: boolean): OpenCrud_OrderBy_Mapping {
     if (depth <= 0) return new Map()
     let properties = getUniversalProperties(model, typeName)
     let m = new Map<string, OrderBy>()
     for (let key in properties) {
         let propType = properties[key].type
-        switch(propType.kind) {
+        switch (propType.kind) {
             case 'scalar':
             case 'enum':
                 if (propType.name != 'JSON') {
-                    m.set(key + '_ASC', {[key]: 'ASC'})
-                    m.set(key + '_DESC', {[key]: 'DESC'})
+                    m.set(key + '_ASC', { [key]: 'ASC' })
+                    m.set(key + '_DESC', { [key]: 'DESC' })
+                    if (properties[key].nullable || nullable) {
```

Comment:
This check makes `nullable` -> `non-nullable` transition a breaking change. I would probably generate sort options regardless nullability.